src/m365/planner/Bucket.ts

```diff
@@ -0,0 +1,7 @@
+export interface Bucket {
```

Comment:
Instead of defining our own type, let's use the type from the `@microsoft/microsoft-graph-types` package